code/_onclick/drag_drop.dm

```diff
@@ -1,28 +1,42 @@
 /*
-	MouseDrop:
-
-	Called on the atom you're dragging.  In a lot of circumstances we want to use the
+	The below procs are called on the atom you're dragging.  In a lot of circumstances we want to use the
 	recieving object instead, so that's the default action.  This allows you to drag
 	almost anything into a trash can.
 */
 
-/atom/proc/CanMouseDrop(atom/over, var/mob/user = usr, var/incapacitation_flags)
-	if(!user || !over)
+/atom/MouseDrop(atom/over)
+	SHOULD_CALL_PARENT(TRUE)
+	if(!can_mouse_drop(over, usr) || !handle_mouse_drop(over, usr))
+		. = ..()
```

Comment:
What does this parent call actually do?

Reply:
Parent call adds the `receive_mouse_drop` timer.

Reply:
Nvm wrong block. Nothing as far as I know, but I figured better to call down in case of future override.